app/routes/labs.missions/route.tsx

```diff
@@ -12,6 +12,7 @@ import { Meatball } from '~/components/meatball/Meatball'
 import type { BreadcrumbHandle } from '~/root/Title'
 
 import fleet from './AstroFleetChart.png'
+import missionTimeline from './mission-timeline.png'
```

Comment:
Where is the source artwork for this graphic? Is it an Adobe Illustrator file? A Python/matplotlib script? Can we get the source artwork under version control, please?

Reply:
I haven't the faintest idea. I took the image from the mock up. @dankocevski do you have any idea how the image is generated?

Reply:
I'm going to open a new issue to get the source material for this graphic under version control. That way we can get this content onto the main branch.

Reply:
https://github.com/nasa-gcn/gcn.nasa.gov/pull/1839/files I have this branch. commenting here for posterity. It has the image, but it's just pulled from the existing mock up website.